micrometer-core/src/main/java/io/micrometer/core/instrument/binder/system/UptimeMetrics.java

```diff
@@ -64,7 +64,7 @@ public void bindTo(MeterRegistry registry) {
 
         TimeGauge.builder("process.start.time", runtimeMXBean, TimeUnit.MILLISECONDS, RuntimeMXBean::getStartTime)
             .tags(tags)
-            .description("Start time of the process since unix epoch.")
+            .description("Start time of the process since unix epoch in seconds.")
```

Comment:
I think adding seconds to the description is redundant and also may be inaccurate behavior. `TimeGauge` is reported in the base unit of registries choice. So, it may be seconds for Prometheus but it is not the case for other registries.

Either this has to be done within the scope of PrometheusMeterRegistry or should be done via a meter filter in the applications.